### ISSUE TITLE
Reading: Temporal Effects on Pre-trained Models for Language Processing Tasks

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#260]
## 0. Paper
 - paper: [[link](https://aclanthology.org/2022.tacl-1.53/)]
 - TACL2022

## 1. What is it?
They evaluate the temporal effects for downstream tasks.

## 2. What is amazing compared to previous works?
They conduct wide range of experiments.

## 3. Where is the key to technologies and techniques?
They use two types of metrics, summary deterioration score (DS) and summary adaptation score (AS)
![スクリーンショット 2023-05-14 23 12 05](https://github.com/a1da4/paper-survey/assets/45454055/e68f51e4-771c-47d5-a29c-0803e8673628)

In the DS score, $a$ is the oldest time period in test datasets:  

```math
DS^{a}_{t} = average(M^{test=2016, ..., 2019}_{train=2014} - M^{test=2015}_{train=2014})
```

On the other hand, in the AS score, $a$ is the oldest train time.

```math
AS^{a}_{t} = average(M^{test=2016, ..., 2019}_{train=2015, ..., 2018} - M^{test=2016, ..., 2019}_{train=2014})
``` 

### 4.1 Temporal deterioration
They used four classification tasks.
![スクリーンショット 2023-05-14 23 46 37](https://github.com/a1da4/paper-survey/assets/45454055/e710e594-7eb8-422e-b60f-11dab630e7f1)

Table 2 shows that:
 - Temporal deterioration occurs in Truecasing and Domain classification tasks
 - Not all tasks suffer from model deterioration
 - Domain adaptation improves the performance in NER, Sentiment, and Domain classification tasks

### 4.2 Without pretraining
They randomly initialized biLSTM.
![スクリーンショット 2023-05-15 0 03 06](https://github.com/a1da4/paper-survey/assets/45454055/724281d4-a9d7-4ab6-bd16-89a0f17cc5af)

Table 3 shows that
 - Temporal deterioration occurs in all tasks
 - Pretraining have significant improvement on many NLP tasks

### 4.3 Additional pretraining vs Self-labeling
They also tested two domain adaptation methods, continual pretraining (without task labels) and self-labeling (fine-tuned at time $s$, then predict labels at time $t$)
![スクリーンショット 2023-05-15 0 23 22](https://github.com/a1da4/paper-survey/assets/45454055/3f8d5410-68ee-485e-848d-7ad21b47adee)

Table 9 shows that self-labeling is effective (same as #259 , fine-tuning is more important than additional pretraining)

**Comment by a1da4** [comment]
#259
